facades/subprojects.gradle

```diff
@@ -1,9 +1,13 @@
 // This magically allows subdirs in this subproject to themselves become sub-subprojects in a proper tree structure
 new File(rootDir, 'facades').eachDir { possibleSubprojectDir ->
-    def subprojectName = 'facades:' + possibleSubprojectDir.name
-    println "Processing facade $subprojectName, including it as a sub-project"
-    include subprojectName
-    def subprojectPath = ':' + subprojectName
-    def subproject = project(subprojectPath)
-    subproject.projectDir = possibleSubprojectDir
+    if (!possibleSubprojectDir.name.startsWith(".")) {
```

Comment:
What about a general approach like this (in the future): https://medium.com/@ungesehn/use-gitignore-for-gradle-task-excludes-e5d011e99f71

`.settings` should be gitignores regardless of where in the file tree...